lib/spring/client/run.rb

```diff
@@ -1,3 +1,4 @@
+require "spring/commands"
```

Comment:
With the current implementation, I think the answer should be yes. However I think that actually we could write this "Running via Spring preloader" message in `Application#serve`, by which point `spring/commands` would be loaded. So I think that may be a better option.